THIRD-PARTY-LICENSES.txt

```diff
@@ -260,7 +261,7 @@ Lists of 357 third-party dependencies.
      (MIT License) JUL to SLF4J bridge (org.slf4j:jul-to-slf4j:1.7.32 - http://www.slf4j.org)
      (Eclipse Public License 1.0) JUnit (junit:junit:4.13.1 - http://junit.org)
      (Apache License 2.0) JVM Integration for Metrics (io.dropwizard.metrics:metrics-jvm:4.1.25 - https://metrics.dropwizard.io/metrics-jvm)
-     (The Apache Software License, Version 2.0) lang-mustache (org.elasticsearch.plugin:lang-mustache-client:7.10.2 - https://github.com/elastic/elasticsearch)
+     (Elastic License 2.0) (Server Side Public License, v 1) lang-mustache (org.elasticsearch.plugin:lang-mustache-client:7.14.1 - https://github.com/elastic/elasticsearch)
```

Comment:
Interesting, this doesn't seem to be an open source license 
https://opensource.org/node/1099

That said, I don't think its viral (as I think we stay open source even if we use non open-source components).

Reply:
it's the main reason why Amazon forked Elasticsearch

Reply:
Yeah, I knew they changed the license for the server to not be open source. 
I just didn't know they made the client libraries not open source as well.